webapp/src/components/App/Studies/CreateStudyDialog.tsx

```diff
@@ -0,0 +1,162 @@
+import debug from "debug";
+import { useSnackbar } from "notistack";
+import { useTranslation } from "react-i18next";
+import { AxiosError } from "axios";
+import { usePromise } from "react-use";
+import * as R from "ramda";
+import { GenericInfo, StudyPublicMode } from "../../../common/types";
+import useEnqueueErrorSnackbar from "../../../hooks/useEnqueueErrorSnackbar";
+import { createStudy } from "../../../redux/ducks/studies";
+import { getStudyVersionsFormatted, getGroups } from "../../../redux/selectors";
+import useAppSelector from "../../../redux/hooks/useAppSelector";
+import useAppDispatch from "../../../redux/hooks/useAppDispatch";
+import FormDialog from "../../common/dialogs/FormDialog";
+import StringFE from "../../common/fieldEditors/StringFE";
+import SelectFE from "../../common/fieldEditors/SelectFE";
+import Fieldset from "../../common/Fieldset";
+import CheckboxesTagsFE from "../../common/fieldEditors/CheckboxesTagsFE";
+import { SubmitHandlerPlus } from "../../common/Form";
+
+const logErr = debug("antares:createstudyform:error");
+
+interface FieldValues {
+  name: string;
+  version: string;
+  publicMode: StudyPublicMode;
+  groups: string[];
+  tags: string[];
+}
+
+interface Props {
+  open: boolean;
+  onClose: VoidFunction;
+}
+
+function CreateStudyDialog(props: Props) {
+  const [t] = useTranslation();
+  const { open, onClose } = props;
+  const { enqueueSnackbar } = useSnackbar();
+  const enqueueErrorSnackbar = useEnqueueErrorSnackbar();
+  const versionList = useAppSelector(getStudyVersionsFormatted);
+  const mounted = usePromise();
+  const dispatch = useAppDispatch();
+
+  const groupList = useAppSelector(getGroups);
+
+  const handleSubmit = async (data: SubmitHandlerPlus<FieldValues>) => {
+    const { name, ...rest } = data.values;
+
+    if (name && name.replace(/\s+/g, "") !== "") {
+      try {
+        await mounted(
+          dispatch(
+            createStudy({
+              name,
+              ...rest,
+            })
+          ).unwrap()
+        );
+
+        enqueueSnackbar(t("studies.success.createStudy", { studyname: name }), {
+          variant: "success",
+        });
+      } catch (e) {
+        logErr("Failed to create new study", name, e);
+        enqueueErrorSnackbar(
+          t("studies.error.createStudy", { studyname: name }),
+          e as AxiosError
+        );
+      }
+      onClose();
+    } else {
+      enqueueSnackbar(t("global.error.emptyName"), { variant: "error" });
+    }
+  };
+
+  const publicModeList: Array<GenericInfo> = [
+    { id: "NONE", name: t("study.nonePublicMode") },
+    { id: "READ", name: t("study.readPublicMode") },
+    { id: "EXECUTE", name: t("study.executePublicMode") },
+    { id: "EDIT", name: t("global.edit") },
+    { id: "FULL", name: t("study.fullPublicMode") },
+  ];
```

Comment:
put this in common/utils/const.ts and remove the translation, instead translate key when using this object).
refacto also PropertiesDialog and NavHeader to use the new const object